src/r2x/parser/reeds.py

```diff
@@ -157,6 +158,26 @@ def _construct_buses(self):
                 )
             )
 
+    def _check_solve_year(self):
+        self.reeds_config = self.config.input_config
+        solve_year = self.reeds_config.solve_year
+        if not solve_year:
+            raise AttributeError("Missing solve year from the configuration class.")
+
+        base_folder = self.config.run_folder
+        input_folder = "inputs_case"
+        modeled_years = "modeledyears.csv"
+
+        modeled_years_file = base_folder / input_folder / modeled_years
+        solve_years_csv = pd.read_csv(modeled_years_file)
```

Comment:
This file is already accessible on the parser data as a polars dataframe (see `self.data['years']`). We can remove the pandas dependency.